prereise/gather/data/remap_ba_area/map_data.py

```diff
@@ -0,0 +1,85 @@
+states_to_map_with_counties = [
+    # NOTE: we skip Alaska and Hawaii
+    "AL",
+    "AR",
+    "AZ",
+    # NOTE: we skip CA because we get its BAAs from other, non-county-sources
+    "CO",
+    "CT",
+    "DE",
+    "FL",
+    "GA",
+    "IA",
+    "ID",
+    "IL",
+    "IN",
+    "KS",
+    "KY",
+    "LA",
+    "MA",
+    "MD",
+    "ME",
+    "MI",
+    "MN",
+    "MO",
+    "MS",
+    "MT",
+    "NC",
+    "ND",
+    "NE",
+    "NH",
+    "NJ",
+    "NM",
+    "NV",
+    # NOTE: we skip NY because we get its BAAs from other, non-county-sources
+    "OH",
+    "OK",
+    "OR",
+    "PA",
+    "RI",
+    "SC",
+    "SD",
+    "TN",
+    "TX",
+    "UT",
+    "VA",
+    "VT",
+    "WA",
+    "WI",
+    "WV",
+    "WY",
+]
+
+miso_name_map = {
+    t[1]: t[0]
+    for t in [
+        ["MISO-0001 ", "LRZ1"],
+        ["MISO-0027 ", "LRZ2_7"],
+        ["MISO-0035 ", "LRZ3_5"],
+        ["MISO-0004 ", "LRZ4"],
+        ["MISO-0006 ", "LRZ6"],
+        ["MISO-8910 ", "LRZ8_9_10"],
+    ]
+}
+
+swpp_name_map = {
+    t[1]: t[0]
+    for t in [
+        ["SWPP-CSWS", "CSWS"],
+        ["SWPP-EDE", "EDE"],
+        ["SWPP-GRDA", "GRDA"],
+        ["SWPP-KACY", "KACY"],
+        ["SWPP-KCPL", "KCPL_INDN"],
+        ["SWPP-LES", "LES"],
+        ["SWPP-MPS", "MPS"],
+        ["SWPP-NPPD", "NPPD"],
+        ["SWPP-OKGE", "OKGE"],
+        ["SWPP-OPPD", "OPPD"],
+        ["SWPP-SECI", "SECI"],
+        ["SWPP-SPRM", "SPRM"],
+        ["SWPP-SPS", "SPS"],
+        ["SWPP-WAUE", "WAUE"],
+        ["SWPP-WFEC", "WFEC"],
+        ["SWPP-WR", "WR"],
+    ]
+}
```

Comment:
Is there any reason we don't define these dictionaries directly, rather than a dict comp over a list?

Reply:
Reformatted to direct dictionaries.